core/__init__.py

```diff
@@ -25,15 +25,41 @@
 """
 
 import logging
+import os
 import webob
+import yaml
 from types import ModuleType
-from typing import Callable, Dict, Union
+from typing import Callable, Dict, Union, List
 from viur.core import session, errors, i18n, request, utils
 from viur.core.config import conf
 from viur.core.tasks import TaskHandler, runStartupTasks
 from viur.core import logging as viurLogging  # Initialize request logging
 
 
+def load_indexes_from_file() -> Dict[str, List]:
```

Comment:
I think this is misplaced here. We should generally try to reduce the length of `__init__` — This should contain only the imports and basic initializations.

Maybe we should move it to the `utils`?

Reply:
Mhh is `utils` the right place for it, because we only use this once in `__init__`.

Reply:
I'm with @ArneGudermann.

Generally, the entire stuff from `__init__.py` should in perspective be moved into something like `route.py` or so.

Reply:
Yes this makes sense to me. But lets make a new PR for this ok ?